packages/ts/react-crud/src/autoform.tsx

```diff
@@ -289,6 +289,13 @@ export function AutoForm<M extends AbstractModel>({
     setShowDeleteDialog(false);
   }
 
+  const handleKeyDown = (event: KeyboardEvent): void => {
+    if (event.key === 'Enter' && (disabled === false || (isEditMode && form.dirty))) {
```

Comment:
`isEditMode` reflects whether an existing item is being edited or not (unfortunate name maybe). So currently it's not possible to submit by enter when creating a new item. I don't think there is any reason to disable this shortcut other than when the form is disabled, or when it's not dirty.

Reply:
`isEditMode` is needed for disabling the Submit button according to the criteria mentioned in https://github.com/vaadin/hilla/issues/1432. 

If we want to keep the `disabled` status of the Submit button consistent with the availability of the form submission via keyboard shortcut, the condition should be something like:
```tsx
event.key === 'Enter' && (disabled === false && (!isEditMode || form.dirty))
```
So probably it makes more sense to define variable for it and use it in both places.